src/javascripts/application-example.mjs

```diff
@@ -0,0 +1,13 @@
+import { initAll } from 'govuk-frontend'
+
+import ExamplePage from './components/example-page.mjs'
+
+// Initialise GOV.UK Frontend
+initAll({
```

Comment:
Example `<iframe>` pages now use **application-example.mjs**

We adjust `initAll()` from GOV.UK Frontend to turn off auto focus